framework/Models/PostProcessors/FTImporter.py

```diff
@@ -89,26 +95,20 @@ def run(self, inputs):
     """
       This method executes the postprocessor action.
       @ In,  inputs, list, list of file objects
-      @ Out, out, dict, dict containing the processed FT
+      @ Out, outputDict, dict, dict containing the processed FT
     """
     faultTreeModel = FTStructure(inputs, self.topEventID)
-    return faultTreeModel.returnDict()
+    outputDict = faultTreeModel.returnDict()
+    outputDict = {'data': outputDict, 'dims':{}}
+    return outputDict
 
-  def collectOutput(self, finishedJob, output):
+  def collectOutput(self, finishedJob, output, options=None):
     """
       Function to place all of the computed data into the output object, (DataObjects)
-      @ In, finishedJob, object, JobHandler object that is in charge of running this postprocessor
+      @ In, finishedJob, object, JobHandler object that is in charge of running this PostProcessor
       @ In, output, object, the object where we want to place our computed results
+      @ In, options, dict, optional, not used in PostProcessor.
```

Comment:
should we specify it since PP doesn't use it?

Reply:
It inherits from the Model base class.